.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: scala
```

Comment:
this is a Travis conf I use in another project, what do you think?
```
jdk: oraclejdk8
language: scala
scala:
- 2.11.8
- 2.12.1
cache:
  directories:
  - "$HOME/.ivy2"
  - "$HOME/.sbt"
  - "$HOME/.m2"
```

Reply:
Makes sense. I've added caching as well.